source/deploy/mobile-app-changelog.md

```diff
@@ -1,12 +1,51 @@
 # Mattermost Mobile Apps Changelog
 
 Latest Mattermost Mobile Apps releases:
-- [1.47.2 Release](#release)
-- [1.47.1 Release](#id1)
-- [1.47.0 Release](#id5)
-- [1.46.0 Release](#id11)
-- [1.45.1 Release](#id19)
-- [1.45.0 Release](#id24)
+- [1.48.0 Release](#release)
+- [1.47.2 Release](#id1)
+- [1.47.1 Release](#id6)
+- [1.47.0 Release](#id11)
+- [1.46.0 Release](#id20)
+- [1.45.1 Release](#id28)
+
+## 1.48.0 Release
+- Release Date: November 16, 2021
+- Server Versions Supported: Server v5.37.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v5.37.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.31 has ended and upgrading to server ESR v5.37.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Highlights
+
+#### Cross-team Recent Mentions
+ - Recent mentions and saved posts now show across all teams.
+
+### Improvements
+ - Adjusted channel override mobile notification preferences for Threads.
+ - Added long-press options to the Global Threads screen.
+ - Added a new messages line to the Threads screen.
+ - Added "pull to refresh" to load threads in the Global Threads screen.
+ - Added OAuth support for plugins and apps.
+ - Added multiselect support for apps forms.
+ - Added a "Rest" field app command to filter empty options on static and dynamic selects and to prohibit executing non-leaf commands.
+ - App bindings now recognize the post menu options for each channel they live in.
+ - Added ``@here`` mention to the ``EnableConfirmNotificationsToChannel`` config setting to show a warning modal when over 5 members might be alerted with ``@here``.
+ - Updated "Jump to..." to match webapp equivalent user interface string.
+
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where image previews could only be viewed once while on the Thread view.
+
+### Known Issues
+ - Channel sidebar disappears sometimes in Airplane mode.
```

Comment:
this is not in airplane mode, this is when the categories are not fetched from the server